iOS/Flows/Conversation/Status View/Reactions/AddReactionView.swift

```diff
@@ -72,14 +63,8 @@ class AddReactionView: UICollectionReusableView {
                                  identifier: nil,
                                  discoverabilityTitle: nil,
                                  attributes: []) { [unowned self] _ in
-                Task {
-                    let controller = ChatClient.shared.messageController(cid: cid, messageId: msg.id)
-                    do {
-                        try await controller.addReaction(with: type)
-                    } catch {
-                        logDebug(error)
-                    }
-                }.add(to: self.taskPool)
```

Comment:
We'll just configure the menu once, send then call a dedicated add reaction function.